### PR TITLE
Use schema.graphql instead of schema.json

### PR DESCRIPTION
**Pull request by kassens**
The `*.graphql` format is easier to read then the JSON serialization.

### CODE REVIEW

File: todo-modern/.gitignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
 .DS_Store
-data/schema.graphql
-__generated__/
\ No newline at end of file
+__generated__/
```
Comment (leebyron) [inline todo-modern/.gitignore]: Should we comment somewhere that typically __generated__ is checked in (or maybe is it actually good to not check it in for smaller projects?)

**Comment by leebyron** [comment]
lgtm

File: todo-modern/.gitignore (hunk 1 above)
Comment (kassens) [inline todo-modern/.gitignore]: Not sure, I think this is something we need to see.

Comment (mjmahone) [inline todo-modern/.gitignore]: I suspect this is very project specific: do you want to treat generated files as a "compilation" step (in which case, gitignore them), or do you want to treat them as source code, so when you git pull your Relay app will work immediately (in which case, check them in)?

Comment (leebyron) [inline todo-modern/.gitignore]: Let's leave this as is because as an example it's nice to have people see the generated files get created

**Comment by leebyron** [comment]
shipit

**Pull request merged**